void-gr-filters.txt

```diff
@@ -152,6 +152,7 @@ zougla.gr/uploads/thomas/travel_city_banner*
 ||ads.naftemporiki.gr
 ||adwebone.com^$third-party
 ||airfasttickets.gr/affiliate/*
+||aj2234.online^
```

Comment:
the relevant URL:
`https://cdn77.aj2234.online/files2234/3/3/22/t/290/gif_fallback_el-GR_300x250_novibet-greek-roulette-novibet-all-bets-blackjack.gif`
is blocked by easylist due to the `_300x250_` included there.

Reply:
I have easylist enabled but it doesn't get blocked for me. IDK why not.
I'll see if I can commit the url ban there.